packages/client/css/specific/modal.css

```diff
@@ -0,0 +1,76 @@
+.Modal {
+  display: block;
+
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  max-width: calc(100% - 48px);
+
+  background: var(--gray-900);
+  border-radius: 4px;
+  overflow: hidden;
+
+  z-index: 950;
+  box-shadow: 0 1px 8px #00000044;
```

Comment:
8-digit hex notation [doesn't seem](https://caniuse.com/#feat=css-rrggbbaa) particularly well supported. I know that browser support isn't priority numero uno, but it seems no worse to use `rgba(0, 0, 0, 0.27)` in this case.